service/src/main/kotlin/xyz/lebkuchenfm/api/commands/CommandsEndpoint.kt

```diff
@@ -14,10 +16,8 @@ private val logger = KotlinLogging.logger {}
 
 fun Route.commandsRouting(commandExecutorService: CommandExecutorService) {
     post("/commands/text") {
-        // TODO: Get UserSession from request when the endpoints get authorization.
-        val session = UserSession("FAKE USER TODO")
-        // val session = call.sessions.get<UserSession>()
-        // checkNotNull(session)
+        val session = call.sessions.get<UserSession>()
+        checkNotNull(session)
```

Comment:
Don't we want to catch it and produce user friendly response?

Reply:
Those endpoints are behind auth strategy. There will always be session here. So realistically `checkNotNull` will never throw. 
If it does a developer made a mistake and that's illegal state, that is not presentable to the end user - it's just internal server error. That will never happen.